src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Client;
+
+use Symfony\Component\BrowserKit\AbstractBrowser;
+use Symfony\Component\HttpFoundation\Response;
+use Webmozart\Assert\Assert;
+
+final class ApiPlatformClient implements ApiClientInterface
+{
+    /** @var AbstractBrowser */
+    private $client;
+
+    /** @var array */
+    private $request;
+
+    /** @var Response|null */
+    private $response = null;
+
+    public function __construct(AbstractBrowser $client)
+    {
+        $this->client = $client;
+    }
+
+    public function index(string $resource): void
+    {
+        $this->response = null;
+        $this->client->request('GET', '/new-api/'.$resource, [], [], ['HTTP_ACCEPT' => 'application/ld+json']);
+    }
+
+    public function buildCreateRequest(string $resource): void
```

Comment:
In the next iteration, maybe client and request could be separate objects (to not make service stateful), and request could be some VO, or created fluent interfaces?